### PR TITLE
ja: Make docs/setup/production-environment/turnkey/icp.md follow v1.17 of the original text

### LINKED ISSUE #22374: ja: Make docs/setup/production-environment/turnkey/icp.md follow v1.17 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/setup/production-environment/turnkey/icp.md to follow v1.17 of the original (English) text.

英文のまま更新するだけで十分です。
翻訳は必須ではありませんが、歓迎します。翻訳用にIssueを作成する必要はありません。

**Why is this needed**
content/ja/docs/setup/production-environment/turnkey/icp.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/blob/dev-1.17-ja.3/content/ja/docs/setup/production-environment/turnkey/icp.md

Original
https://github.com/kubernetes/website/blob/release-1.17/content/en/docs/setup/production-environment/turnkey/icp.md

diff between translated and v1.17
https://github.com/kubernetes/website/commit/e66b9a542ad90738753288cc767655454c6d6a99

**Note**
Currently, we use **dev-1.17-ja.3** branch. Please open a PR targeting the branch.

ref
[How To Contribute](https://kubernetes-docs-ja.kibe.la/shared/entries/c5878aa5-ad1f-4f29-a5bb-25853cbc14ec)
[翻訳スタイルガイド](https://kubernetes-docs-ja.kibe.la/shared/entries/5efe4fa7-d2a1-4a1d-8bc3-ce7ccdc064a6)

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel

### PR DESCRIPTION
**Pull request by hiyokotaisa**
Close #22374

### CODE REVIEW

File: content/ja/docs/setup/production-environment/turnkey/icp.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ IBM Cloud Private can also run on the AWS cloud platform by using Terraform. To
 
 ## Azure上でのIBM Cloud Private
 
-You can enable Microsoft Azure as a cloud provider for IBM Cloud Private deployment and take advantage of all the IBM Cloud Private features on the Azure public cloud. For more information, see [IBM Cloud Private on Azure](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.2/supported_environments/azure_overview.html).
+You can enable Microsoft Azure as a cloud provider for IBM Cloud Private deployment and take advantage of all the IBM Cloud Private features on the Azure public cloud. For more information, see [IBM Cloud Private on Azure](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.2.0/supported_environments/azure_overview.html).
 
 ## Red Hat OpenShift上でのIBM Cloud Private
```
Comment (inductor) [inline content/ja/docs/setup/production-environment/turnkey/icp.md:40]:
```suggestion
## Red Hat OpenShiftを用いたIBM Cloud Private
```

**Comment by nasa9084** [comment]
inductorさんのsuggest以外はlgtmです

**Pull request merged**